pycodestyle.py

```diff
@@ -56,6 +56,15 @@
 import time
 import tokenize
 import warnings
+try:
+    from functools import lru_cache
+except ImportError:
+    def lru_cache(maxsize=128):  # noqa as it's a fake implementation.
```

Comment:
What versions of Python are we guarding against here? I think the minimum version we support now is 3.3.